gen/generator.toml

```diff
@@ -91,7 +91,20 @@ output_ignorelist = [
                      "UA_INT64_MIN", 
                      "UA_UINT64_MAX", 
                      "UA_UINT64_MIN", 
-                     "UA_PRINTF_GUID_FORMAT"
+                     "UA_PRINTF_GUID_FORMAT",
```

Comment:
For some reason, these identifiers needed to be added to the ignorelist while re-generating the wrapper. Do you have a clue why this has not been an issue before?

Reply:
I think these identifiers are OS specific. I always ran the generator on Windows. Are you by any chance on a Linux/Unix system?

Reply:
Yes, that could be possible as I ran the generator on Linux.

Reply:
I think this is fine then. 

There are Windows specific identifiers that I needed to add to the ignore list when running it on Windows. When you are running the generator on Linux, Clang.jl won't try to insert these. 

The situation for you on Linux is just the other way around. 

I think by adding these, it makes the generator more robust, as it will be able to run on both OS (I have a macbook that I can try with as well).